thirdparty/cJSON/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 project(cJSON)
 
 add_library(cJSON STATIC cJSON.c)
+target_include_directories(cJSON
+  PUBLIC
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+    $<INSTALL_INTERFACE:include/smtk/${SMTK_VERSION}>)
```

Comment:
Should we be installing to `include/smtk/${SMTK_VERSION}/third-party/cJSON` instead?